pages/api/augment-request.ts

```diff
@@ -161,7 +165,10 @@ async function getOracleRequestPrices(
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
         identifier: event?.args?.identifier,
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-        time: event?.args?.timestamp,
+        time:
+          event?.args?.timestamp instanceof BigNumber
+            ? event?.args?.timestamp?.toNumber()
+            : event?.args?.timestamp,
```

Comment:
somehow the events are different, sometimes its a number sometimes its a big number, this will make that check